code/game/atoms.dm

```diff
@@ -65,6 +65,7 @@
 	/// A luminescence-shifted value of the last color calculated for chatmessage overlays
 	var/chat_color_darkened
 
+	var/list/hud_possible //HUD images that this mob can provide.
```

Comment:
Please use the autodoc format for this particular comment, of course also changing the `mob` aspect of it.

Reply:
This is done.